database/migrations/2025_07_15_171455_create_airlines_table.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('airlines', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->string('business_description');
```

Comment:
```suggestion
            $table->string('description');
```
Just "description" is fine.